src/graphics/aurora/model_kotor.h

```diff
@@ -93,9 +95,14 @@ class ModelNode_KotOR : public ModelNode {
 	void load(Model_KotOR::ParserContext &ctx);
 
 private:
+	uint32 _mdxStructSize;
+	uint16 _vertexCount;
+	uint32 _offNodeData;
```

Comment:
These arent used past the parsing of the model and move to the ParserContext

Reply:
Fixed